node/src/main/scala/coop/rchain/node/diagnostics/package.scala

```diff
@@ -216,6 +218,14 @@ package object diagnostics {
             case c: metric.Histogram => c.record(value, count)
           }
         }
+
+      def startTimer(name: String): F[MetricsTimer] =
+        Capture[F].capture {
+          val t = Kamon.timer(name).start()
+          new MetricsTimer {
+            def stop(): Unit = t.stop()
```

Comment:
`t.stop` is effectful, should be closed over `Task`

Reply:
I'm not sure I understand what you mean.  I know that I could have used `t.stop` instead of `t.stop()` here. Both @Sebastion and I agreed that this made the code more clear.

What do you mean by closed over `Task`?

Reply:
what i meant was t.stop is effectful, should be closed over F

```scala
def stop: F[Unit] = Capture[F].caputre(t.stop)
```